spyder/app/tests/test_mainwindow.py

```diff
@@ -105,6 +105,49 @@ def close_window():
 # Tests
 #==============================================================================
 @flaky(max_runs=3)
+def test_change_cwd_ipython_console(main_window, qtbot, tmpdir):
+    """
+    Test synchronization with working directory and File Explorer when changing Ipython console.
```

Comment:
... when changing *cwd* in *the IPython* console.

Also, make this docstring fit in two lines instead of one (it's too long).